Apps/W1/Email - SMTP Connector/app/src/SMTPConnectorImpl.Codeunit.al

```diff
@@ -183,7 +183,17 @@ codeunit 4513 "SMTP Connector Impl." implements "Email Connector"
         AttachmentInStream: InStream;
     begin
         // From name/email address
-        SMTPMessage.AddFrom(SMTPAccount."Sender Name", SMTPAccount."Email Address");
+        if EmailMessage.GetFromAddress() <> '' then
+            FromAddress := EmailMessage.GetFromAddress()
+        else
+            FromAddress := SMTPAccount."Email Address";
+
+        if EmailMessage.GetFromName() <> '' then
+            FromName := EmailMessage.GetFromName()
+        else
+            FromName := SMTPAccount."Sender Name";
+
+        SMTPMessage.AddFrom(FromName, FromAddress);
```

Comment:
Do we also want to update other 1st part connectors to respect the new fields?

Reply:
Add a modification to codeunit 4509 "Email - Outlook API Helper"